### DISCUSSION TITLE
Is there any way to unblock bookmarklets?

### DISCUSSION THREAD

**Discussion by Mirkwood815** [3kh0/ext-remover#672] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Like the title says. Ive tried a bunch of shit and Im stuck here.

**Comment by Hilololsusy** [comment]
If you ahve inspect element, it works roughly 50% of the time. Other than that, I ahve no idea

**Comment by ILikeBypassing** [comment, marked as answer]
no

**Comment by AshtonDavies** [comment]
The best you can do is make a DNS server, but nginx setup for that purpose is hard but it wouldn't work for Point-Blank if that's what you're trying to do, so other than that, you can't run your own scripts.

**Comment by thebossjeff** [comment]
As of yesterday, my friend and I found a very obscure way to run javascript without triggering the actual javascript block.
I am currently working on making this into a usable thing.

**Reply by thebossjeff** [reply]
I have gotten it to run the extension remover, with the caveat that it reloads the page when it runs, therefore rendering the current LTBEEF exploit unusable, for the present.

**Reply by YeeticusPrimed** [reply]
can i have the obscure way idc if its hard or obscure i need ext launcher

**Reply by Aka-but-nice** [reply]
Ah, I may be wrong, but, it appears that you are not being patient.
